iwyu_location_util.h

```diff
@@ -37,8 +37,8 @@
 // file path, line number, column number, and which file (if any)
 // #includes the file.
 
-#ifndef DEVTOOLS_MAINTENANCE_INCLUDE_WHAT_YOU_USE_IWYU_FILE_UTIL_H_
-#define DEVTOOLS_MAINTENANCE_INCLUDE_WHAT_YOU_USE_IWYU_FILE_UTIL_H_
+#ifndef INCLUDE_WHAT_YOU_USE_IWYU_LOCATION_UTIL_H_
+#define INCLUDE_WHAT_YOU_USE_IWYU_LOCATION_UTIL_H_
```

Comment:
I did so, but I got during push.

git push MineIncludeWhatYouUse ClangTidyWarningsCleanup:ClangTidyWarningsCleanup
To https://github.com/EugeneZelenko/include-what-you-use
 ! [rejected]        ClangTidyWarningsCleanup -> ClangTidyWarningsCleanup (non-fast-forward)
error: failed to push some refs to 'https://github.com/EugeneZelenko/include-what-you-use'
hint: Updates were rejected because a pushed branch tip is behind its remote
hint: counterpart. Check out this branch and integrate the remote changes
hint: (e.g. 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.

I think it'll be better if somebody who knows git better then me will perform merge.